suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -101,12 +101,12 @@ export const TransactionList = ({
 
     return (
         <Box style={applyStyle(listWrapperStyle)}>
-            <SectionList
-                sections={sectionsData}
-                renderSectionHeader={renderSectionHeader}
-                renderItem={renderItem}
-                ListHeaderComponent={listHeaderComponent}
+            <FlatList
```

Comment:
This should stay SectionList. Month is one section, transaction is one item. If you will have users with lot of transaction per month this will be slow for them.

Reply:
fixed in [30b5f1a](https://github.com/trezor/trezor-suite/pull/7392/commits/30b5f1a27811ed560da7091928ce739d439d0ca9)